htdocs/expensereport/class/expensereport.class.php

```diff
@@ -619,6 +619,7 @@ public function fetch($id, $ref = '')
 		$sql .= " FROM ".MAIN_DB_PREFIX.$this->table_element." as d";
 		if ($ref) {
 			$sql .= " WHERE d.ref = '".$this->db->escape($ref)."'";
+			$sql .= " WHERE d.entity IN (".getEntity('expensereport').")";
```

Comment:
I think you want to do 
AND
and not
WHERE

Reply:
Damn.... Of course